### PR TITLE
Update CHANGELOG.md

### PR DESCRIPTION
**Pull request by bschnurr**

### CODE REVIEW

**Review by heejaechang — APPROVED** [review]

**Review by debonte — APPROVED** [review]

**Comment by bschnurr** [comment]
new 'python.analysis.addImport.exactMatchOnly'

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,23 @@
 # Changelog
 
+## 2023.2.21 (8 February 2023) PreRelease
+
+Notable changes:
+
+-   Bug fix: Empty kw only dataclass has incorrect type hint for `__init__` signature [pylance-release#3916](https://github.com/microsoft/pylance-release/issues/3916)
+-   Bug fix: Docstring of @property is not rendered correctly [pylance-release#3888](https://github.com/microsoft/pylance-release/issues/3888)
+-   Bug fix: Pandas Dataframe.explode function parameter TypeError [pylance-release#3871](https://github.com/microsoft/pylance-release/issues/3871)
```
[the remark this answers is not in the document]
Comment (bschnurr) [reply CHANGELOG.md:9]: oops. forgot to update the stubs

[the remark this answers is not in the document]
Comment (bschnurr) [reply CHANGELOG.md:9]: i'll remove it from the notes.

Comment (bschnurr) [inline CHANGELOG.md:9]:
```suggestion
```

**Review by rchiodo — APPROVED** [review]

**Pull request merged**